RELEASE_STEPS.md

```diff
@@ -40,6 +40,7 @@ Do changes for a release:
 - Add an entry to ChangeLog, remember to change the day of the week in the release date
   - For major and minor (prior 1.0.0) releases mention API and ABI compatibility with the previous release
 - For major and minor releases, update `UMF_VERSION_CURRENT` in `include/umf/base.h` (the API version)
+  - For changes in ops structures, update corresponding UMF_*_OPS_VERSION_CURRENT
```

Comment:
you could also update our PR template

EDIT: hmm should we do this only once per release...? if so, please ignore my comment

Reply:
added